--- a/bin/make-database
+++ b/bin/make-database
@@ -45,15 +45,15 @@ DROP TABLE IF EXISTS checkins;
 CREATE TABLE checkins (
   type enum('Change','Add','Remove'),
   ci_when datetime DEFAULT '0000-00-00 00:00:00' NOT NULL,
-  whoid mediumint(9) DEFAULT '0' NOT NULL,
-  repositoryid mediumint(9) DEFAULT '0' NOT NULL,
-  dirid mediumint(9) DEFAULT '0' NOT NULL,
-  fileid mediumint(9) DEFAULT '0' NOT NULL,
+  whoid mediumint(9)  NOT NULL,
+  repositoryid mediumint(9)  NOT NULL,
+  dirid mediumint(9)  NOT NULL,
+  fileid mediumint(9)  NOT NULL,
   revision varchar(32) binary DEFAULT '' NOT NULL,
   stickytag varchar(255) binary DEFAULT '' NOT NULL,
-  branchid mediumint(9) DEFAULT '0' NOT NULL,
-  addedlines int(11) DEFAULT '0' NOT NULL,
-  removedlines int(11) DEFAULT '0' NOT NULL,
+  branchid mediumint(9)  NOT NULL,
+  addedlines int(11)  NOT NULL,
+  removedlines int(11)  NOT NULL,
   descid mediumint(9),
   UNIQUE repositoryid (repositoryid,dirid,fileid,revision),
   KEY ci_when (ci_when),
@@ -68,7 +68,7 @@ DROP TABLE IF EXISTS descs;
 CREATE TABLE descs (
   id mediumint(9) NOT NULL auto_increment,
   description text,
-  hash bigint(20) DEFAULT '0' NOT NULL,
+  hash bigint(20)  NOT NULL,
   PRIMARY KEY (id),
   KEY hash (hash)
 ) TYPE=MyISAM;
@@ -107,10 +107,10 @@ CREATE TABLE repositories (
 
 DROP TABLE IF EXISTS tags;
 CREATE TABLE tags (
-  repositoryid mediumint(9) DEFAULT '0' NOT NULL,
-  branchid mediumint(9) DEFAULT '0' NOT NULL,
-  dirid mediumint(9) DEFAULT '0' NOT NULL,
-  fileid mediumint(9) DEFAULT '0' NOT NULL,
+  repositoryid mediumint(9)  NOT NULL,
+  branchid mediumint(9)  NOT NULL,
+  dirid mediumint(9)  NOT NULL,
+  fileid mediumint(9)  NOT NULL,
   revision varchar(32) binary DEFAULT '' NOT NULL,
   UNIQUE repositoryid (repositoryid,dirid,fileid,branchid,revision),
   KEY repositoryid_2 (repositoryid),
